docs/developers/celestia-node.md

```diff
@@ -13,16 +13,17 @@ cd $HOME
 rm -rf celestia-node
 git clone https://github.com/celestiaorg/celestia-node.git
 cd celestia-node/
+git checkout tags/v0.3.0-rc1
```

Comment:
It might be worth it to use a variation of https://github.com/celestiaorg/docs/blob/e0f76b2067ce50bc6f29eec07495f92a45274f08/docs/developers/celestia-app.md?plain=1#L18-L20

Reply:
did it

Reply:
You used `app` instead of `node`.

Reply:
whoops, long day i guess